### PR TITLE
OH2-222 | Align with they latest api spec file and add ward in visit form and table

### PR DESCRIPTION
**Pull request by SteveGT96**
See OH2-222.

### CODE REVIEW

**Review by mwithi — CHANGES_REQUESTED** [review]
It seems ok, but please remove also the "+ CREATE NEW OPD VISIT” button

**Comment by mwithi** [comment]
Up!

**Comment by mwithi** [comment]
I think header is ok (we should think about how to change the logo though)

image

pay attention that long notes are not wrapped and scrollbars should be avoided, as well as untranslated labels (see _angal.lab.blood.txt_)

image

**Comment by mwithi** [comment]
Also, seems there are conflicts with `src/styles/index.scss`

File: src/generated/runtime.ts
Comment (mwithi) [inline src/generated/runtime.ts:1]: `withCredentials` has to stay

File: src/generated/apis/LoginApiApi.ts
Comment (mwithi) [inline src/generated/apis/LoginApiApi.ts:1]: why so many files are deleted?

Comment (SteveGT96) [reply src/generated/apis/LoginApiApi.ts:1]: When I was working on the task, I replaced the api file with the one provided by the backend.

**Pull request merged**